### PR TITLE
Feat/migrate pages to content hub

### PR DESCRIPTION
**Pull request by tkrch**
### Motivation

Continuation of #260 , #278

### CODE REVIEW

**Comment by farmergeek94** [comment]
@tkrch While this is working good, there is one extra bit of info that is still coming in.  The content types are still getting assigned to the channel via the table [CMS_ContentTypeChannel].  This allows them to still be added as web page items on the imported channel.

**Comment by bluemodus-mwills** [comment]
Hi @tkrch & @farmergeek94 -- @twozero6, @bluemodus-gfuller , and I are working on a migration right now, and would like to use this feature. Is it safe for us to work off this branch, or should we wait? Does the time it's taking on review mean there are known issues or changes are coming?

**Comment by bluemodus-mwills** [comment]
Thank you @tkrch !

**Comment by farmergeek94** [comment]
Ok that makes sense.  We can always fork that feature branch if we have to as well.

File: Migration.Tool.Extensions/ClassMappings/ClassMappingSample.cs [hunk 1]
```diff
@@ -10,6 +10,61 @@ namespace Migration.Tool.Extensions.ClassMappings;
 
 public static class ClassMappingSample
 {
+    public static IServiceCollection AddReusableRemodelingSample(this IServiceCollection serviceCollection)
```
Comment (akfakmot) [inline Migration.Tool.Extensions/ClassMappings/ClassMappingSample.cs:13]: New sample method without reference in README

**Comment by twozero6** [comment]
@akfakmot I've been working in this branch and I ran into a bit of an issue. 

{72FFD4EB-B7FD-44A0-8B68-BB23984A4101}

When I hit line 227, f is null and throws an exception. Even if I swap the fields around, it's always the last field that has the issue. I did a bit of digging and found that the form definition on line 139 in MigratePageTypesCommandHandler.cs is always missing the last field. It also has some unexpected things as if the form definition from the source was merged with the new one. I did make sure that my field names were unique. Is this something you could look into?

Also, I can start a new issue for this if you think it's appropriate. Is it possible to get the Node context in the ConvertFrom method? For example, I would like to get the node's NodeGUID so that I can do additional lookups (get children) before setting the value.

**Review by akfakmot — APPROVED** [review]

**Pull request merged**